packages/components/src/input/src/Input.css

```diff
@@ -148,7 +148,7 @@
 .o-ui-input.o-ui-input-invalid:not(.o-ui-input-focus):not(.o-ui-input-disabled):hover,
```

Comment:
I'm guessing you followed the focus style for the rich text editor and not the input since the design does not have a plain textarea. I find it weird that the focus style does not have the hover background color like the other input. Maybe we could validate with design?
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/6cef3d74-7c57-4490-bfca-5d376d4a04de)

Reply:
I copied the textinput since they are the most similar. Igloo's textarea is the same. Focus/Active is a bit awkward for input, since focusing the textinput/area automatically make it "active" resulting in the active state being white.